router-tests/query_plans_test.go

```diff
@@ -168,4 +168,30 @@ func TestQueryPlans(t *testing.T) {
 			}
 		})
 	})
+	t.Run("include operation name in each request", func(t *testing.T) {
+		testenv.Run(t, &testenv.Config{
+			ModifyEngineExecutionConfiguration: func(cfg *config.EngineExecutionConfiguration) {
+				cfg.Debug.AlwaysIncludeQueryPlan = true
+				cfg.EnableOperationNameToSubgraphPropagation = true
```

Comment:
I still think this name over complicates it 👀 .

`EnableFetchOperationName` 
`EnableSubgraphFetchOperationName `

?